src/RpcServer/RpcServer.Utilities.cs

```diff
@@ -13,14 +13,14 @@ partial class RpcServer
         private JObject ListPlugins(JArray _params)
         {
             return new JArray(Neo.Plugins.Plugin.Plugins
-                .OrderBy(u => u.Name)
+                .OrderBy(u => u.Name).Where(u => !(u is ILogPlugin))
                 .Select(u => new JObject
                 {
                     ["name"] = u.Name,
                     ["version"] = u.Version.ToString(),
                     ["interfaces"] = new JArray(u.GetType().GetInterfaces()
                         .Select(p => p.Name)
-                        .Where(p => p.EndsWith("Plugin"))
+                        .Where(p => p.EndsWith("Plugin") || p.EndsWith("Provider"))
```

Comment:
This seems that we will have a lot of || in a future, we should change this logic

Reply:
Why do we need to filter it? I think we can just list all the interfaces.

Reply:
> Why do we need to filter it? I think we can just list all the interfaces.

do you want IDisposable to show in all plugins?